docs/core/install/linux.md

```diff
@@ -40,6 +40,12 @@ You may need to install [.NET dependencies](https://github.com/dotnet/core/blob/
 
 .NET is [supported by Microsoft](https://github.com/dotnet/core/blob/main/microsoft-support.md) when downloaded from a Microsoft source. Best effort support is offered from Microsoft when downloaded from elsewhere. You can open issues at [dotnet/core](https://github.com/dotnet/core) if you run into problems.
 
+## Important environment settings
+
+The `DOTNET_ROOT` environment variable helps apps discover where .NET SDKs and runtimes are installed. This variable isn't required when .NET is installed to the default directory, for example, `/usr/lib/dotnet` on Ubuntu. When multiple versions of .NET are installed to different directories, set `DOTNET_ROOT` to the .NET folder you want to use. Additionally, if you want to use that version's `dotnet` command, ensure that there's a symbolic link in the `/usr/bin/` folder for the `dotnet` command, or the folder is included in the `PATH` variable.
```

Comment:
I do not think that this belongs to this landing page. People are likely going to land on this page when they search for how to install .NET on Linux. We did not tell them how to do that yet, and we are pushing `DOTNET_ROOT` malaises on them. `DOTNET_ROOT` is for more advanced use cases only. 95% of .NET users should not ever know that it exists.

> This variable isn't required when .NET is installed to the default directory, for example, `/usr/lib/dotnet` on Ubuntu.

This is not an accurate statement. `DOTNET_ROOT` is only needed for some cases of manual and scripted installations. It is not needed for installations from package repositories. 

For installations from package repository, the default installation directory depends on the package feed. The package installer is expected to configure the installation directory via a file under `/etc/dotnet/install_location_...`. There were some bugs around this, but they should be all fixed now.